src/io/flutter/view/FlutterView.java

```diff
@@ -252,8 +251,6 @@ private void addInspectorViewContent(FlutterApp app, @Nullable InspectorService
         addDisabledTab(WIDGET_TAB_LABEL, runnerTabs, app, toolbarGroup);
         addDisabledTab(RENDER_TAB_LABEL, runnerTabs, app, toolbarGroup);
       }
-
-      addPerformanceTab(runnerTabs, app, !hasInspectorService);
```

Comment:
Maybe we should temporarily add a placeholder tab that just has a link to open the new perf tab?

Reply:
Sure, I can add that. Probably would be good to have a lead for existing users.